### PR TITLE
add appveyor configuration

### PR DESCRIPTION
**Pull request by proppy**
for windows build test.

### CODE REVIEW

**Comment by proppy** [comment]
rebased

**Comment by proppy** [comment]
@claywilkinson this is green on windows now, can you review?

File: gles3jni/app/build.gradle [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    }
+    android.buildTypes {
+        release {
+            minifyEnabled = false
+            proguardFiles  += file('proguard-rules.txt')
+        }
+    }
+
+    android.productFlavors {
+        create ("arm7") {
```
Comment (claywilkinson) [inline gles3jni/app/build.gradle]: no 64 bit?

**Comment by claywilkinson** [comment]
LGTM - one question why are you using launchMode=singleTask?

**Pull request merged**